R/derive_param_computed.R

```diff
@@ -241,3 +241,45 @@ derive_derived_param <- function(dataset,
 
   bind_rows(dataset, hori_data)
 }
+
+#' Adds a Parameter Computed from the Analysis Value of Other Parameters
+#'
+#' @description
+#' `r lifecycle::badge("deprecated")`
+#'
+#' This function is deprecated.  Please use `derive_param-computed()` instead.
+#'
+#' @inheritParams derive_param_computed
+#'
+#' @author Stefan Bundfuss
+#'
+#' @return The input dataset with the new parameter added. Note, a variable will only
+#'    be populated in the new parameter rows if it is specified in `by_vars`.
+#'
+#' @keywords derivation bds
+#'
+#' @export
+#'
+derive_derived_param <- function(dataset,
+                                 by_vars,
+                                 parameters,
+                                 analysis_value,
+                                 set_values_to,
+                                 filter = NULL,
+                                 constant_by_vars = NULL,
+                                 constant_parameters = NULL) {
+  ### BEGIN DEPRECATION
```

Comment:
Please remove the begin and end of deprecation comments. That doesn't make a lot of case in this context (compared to when you deprecate a single parameter of an existing function).